scene/animation/animation_node_state_machine.h

```diff
@@ -33,6 +33,7 @@
 
 #include "core/math/expression.h"
 #include "scene/animation/animation_tree.h"
+#include "scene/main/window.h"
```

Comment:
Is this include required?

Reply:
Its needed because `get_root()` returns a Window, but I could move it to the `.cpp` file instead